Marsiple/Common/Network/HTTPResponseCodes.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  HTTPResponseCodes.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 27/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+enum HTTPResponseCodes: Int {
```

Comment:
The name should be `HTTPResponseCodes`-> `HTTPResponseCode`. When naming enums, it's more appropriate and correct to use a singular instead of plural, as you're modelling an entity, not entities.

Furthermore, `case naming`; I would name these cases using their fully qualified names, rather than their HTTP code (which is already used at the raw value for these cases). 
In example, `code404` would become `notFound`. This way we provide all of the contextual information by setting descriptive names and associating these names with the correct HTTP code values.

Also, what I would do here is to eject the `static func getMessage(forCode code: Int) -> String` method and use `HTTPResponseCode(rawValue: value).message` whenever we need a message based on the HTTP code. This is a `failable initialiser`, so if we wanted to detect the case where we have an unknown error code, we could just check whether the resulting `instance` is `nil` or not.